src/experiments.ts

```diff
@@ -0,0 +1,182 @@
+import { configstore } from "./configstore";
+import * as clc from "colorette";
+import { FirebaseError } from "./error";
+
+export interface Experiment<Name extends string> {
+  name: Name;
+  shortDescription: string;
+  fullDescription?: string;
+  public?: boolean;
+  docsUri?: string;
+  default?: boolean;
+}
+
+function experiment<Name extends string>(
+  name: Name,
+  shortDescription: string,
+  opts?: {
+    fullDescription?: string;
+    public?: boolean;
+    docsUri?: string;
+    default?: boolean;
+  }
+): Experiment<Name> {
+  return {
+    ...opts,
+    name,
+    shortDescription,
+  } as Experiment<Name>;
+}
+
+export const EXPERIMENTS = Object.freeze([
+  // meta:
+  experiment("experiments", "enables the experiments family of commands"),
+
+  // Realtime Database experiments
+  experiment("rtdbrules", "TODO"),
+  experiment("rtdbmanagement", "TODO"),
+
+  // Extensions experiments
+  experiment("ext", "Enables the 'ext:sources:create' command"),
+  experiment("extdev", "Enalbes the 'ext:dev' family of commands"),
+
+  // Cloud Functions for Firebase experiments
+  experiment("pythonfunctions", "Python support for Cloud Functions for Firebase", {
+    fullDescription:
+      "Adds the ability to initializea and deploy Cloud " +
+      "Functions for Firebase in Python. While this feature is experimental " +
+      "breaking API changes are allowed in MINOR API revisions",
+  }),
+  experiment("golang", "Golang support for Cloud Functions for Firebase. Does not work.", {
```

Comment:
@taeold I wonder if we should just delete this code path for now. WDYT? Maybe we can keep the folder but not load the runtime delegate in the main list?

Reply:
Chose to delete